src/plumbing/options/mod.rs

```diff
@@ -512,10 +512,17 @@ pub mod remote {
 }
 
 pub mod mailmap {
+    use gix::bstr::BString;
+
     #[derive(Debug, clap::Subcommand)]
     pub enum Subcommands {
         /// Print all entries in configured mailmaps, inform about errors as well.
         Entries,
+        /// Print all entries in configured mailmaps, inform about errors as well.
```

Comment:
The comment here is incorrect.

Reply:
Right!
Can you make a suggestion? I would just apply it then.

Reply:
Perhaps:

```suggestion
        /// Print the canonical form of contacts according to the configured mailmaps.
```

Reply:
Perhaps:

```suggestion
        /// Print the canonical form of contacts according to the configured mailmaps.
```